### PR TITLE
 Remove CURLY_STUFF hack 

### PR DESCRIPTION
**Pull request by pablogsal**

### CODE REVIEW

**Comment by pablogsal** [comment]
Note: This PR is rebased on top of #39 (it has some commits of that PR).

File: pegen/grammar_parser.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ class GeneratedParser(Parser):
 
     @memoize
     def start(self):
-        # start: rules $ { Grammar ( rules ) }
+        # start: rules $ {'Grammar ( rules )'}
```
Comment (gvanrossum) [inline pegen/grammar_parser.py]: I don't like this change -- there must be something in the generator that causes this change.

File: pegen/metagrammar.gram [hunk 2]
```diff
@@ -54,4 +54,17 @@ atom:
     | NAME {NameLeaf(name.string) }
     | STRING {StringLeaf(string.string)}
 
-action: action=CURLY_STUFF {action}
+# Mini-grammar for the actions
```
Comment (gvanrossum) [inline pegen/metagrammar.gram]: Reminder to update TODO.md.

File: pegen/metagrammar.gram [hunk 3]
```diff
@@ -54,4 +54,17 @@ atom:
...
+
+action: "{" stuffs "}" { stuffs }
+
+stuffs:
```
Comment (gvanrossum) [inline pegen/metagrammar.gram]: Can you think of better names than `stuffs` and `stuff`? I meant them to be like `foo` and `bar` but never got around to inventing better names. Now's the time, I guess. :-)

Comment (pablogsal) [reply pegen/metagrammar.gram]: I updated the PR to use the uninspired `action_items`  and `action_item`. I was thinking about something with `atom`/`molecule` but I was unsure if it fits the convention with those. 

Please, advice if you find better names :)

**Pull request merged**